app/views/static_pages/tools_resources.html.erb

```diff
@@ -9,7 +9,7 @@
   <p>
     Watch the short video and download the playbook to get started today.
   </p>
-  <%= link_to 'Download Pdf', pdf_path(@pdf), class: "button" %>
+  <%= link_to 'Download Pdf', pdf_path(@pdf_learn_more), class: "button" if @pdf_learn_more %>
 </div>
```

Comment:
I like that the button is no longer displayed if no pdf is updload! Nice! :space_invader: